pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -440,23 +451,127 @@ func (r *ReconcileUnifiedPushServer) Reconcile(request reconcile.Request) (recon
 	}
 	//#endregion
 
-	//#region Postgres DeploymentConfig
-	postgresqlDeploymentConfig, err := newPostgresqlDeploymentConfig(instance)
+	//#region MIGRATION from old resources to new ones
+
+	// TODO: This migration block should be removed after a major release!
+	// TODO: in UPS operator version 0.x, we were using DCs and ImageStreams.
+	// TODO: in 1.0.0, we introduced this code block to migrate from old resources to new ones.
+	// TODO: in 2.0.0, we should get rid of this migration block, as well as unneeded permissions
+	// TODO: to access these old resources.
```

Comment:
@laurafitzgerald see the comments here

Reply:
Looks good @aliok Question: Should the 0.x version be a semver?